src/test/java/uk/gov/companieshouse/overseasentitiesapi/utils/NaturesOfControlCollectionBuilderTest.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.companieshouse.overseasentitiesapi.utils;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.companieshouse.overseasentitiesapi.model.NatureOfControlJurisdictionType;
+import uk.gov.companieshouse.overseasentitiesapi.model.NatureOfControlType;
+
+import java.util.Arrays;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+class NaturesOfControlCollectionBuilderTest {
```

Comment:
Blank line should be added before the `class` definition.

Reply:
Added